skipruntime-ts/core/README.md

```diff
@@ -0,0 +1,20 @@
+# @skipruntime/core
+
+This is the Skip runtime public API: types and operations that can be used to
+write and interact with reactive computations.
+
+See the [docs](https://skiplabs.io/docs/api/api) for more details.
```

Comment:
At next docs site update this will be:
```suggestion
See the [docs](https://skiplabs.io/docs/api/core) for more details.
```

Reply:
This one was intentional -- the suggested link 404s

Reply:
It won't once https://github.com/SkipLabs/skip/pull/720 lands, and api/api will 404 at that point.